src/zkevm_specs/state.py

```diff
@@ -816,10 +824,19 @@ def _mock_mpt_updates(ops: List[Operation], randomness: FQ) -> Dict[Tuple[FQ, FQ
         if mpt_key is None or mpt_key in mpt_map:
             continue
 
+        field_tag = op.field_tag
+        proof_type = MPTProofType.IsStorageMod  # type warning if None
+        if field_tag == AccountFieldTag.Nonce:
+            proof_type = MPTProofType.IsNonceMod
+        elif field_tag == AccountFieldTag.Balance:
+            proof_type = MPTProofType.IsBalanceMod
+        elif field_tag == AccountFieldTag.CodeHash:
+            proof_type = MPTProofType.IsCodeHashProof
```

Comment:
If you write this multiple times, you should be able to define a class method for MPTProofType which recieves the fieldtag or similar and returns the enum variant that corresponds.

On this way you don't need to keep re-writing the `if-elif` block again and again